src/components/Header/index.js

```diff
@@ -7,9 +7,28 @@ class Header extends React.Component {
 
   constructor(props) {
     super(props);
+
+    this.state = {
+      embed: false
+    };
+  }
+
+  componentDidMount() {
+    !this.state.embed ? this.isEmbed() : '';
```

Comment:
You'd probably better use:
```javascript
this.state.embed || this.isEmbed();
```

But the main doubt is what you want to achieve: do you want to check if the URL contains `embed` when your state say it's not embedded? If so, write a comment to make it clearer or change the name of the function to something like `checkIfEmbed`. And you could move that line inside the function so `componentDidMount` would look like:

```javascript
componentDidMoun() {
   this.checkIfEmbed();
}
```